Lib/ctypes/__init__.py

```diff
@@ -338,6 +338,16 @@ def __init__(self, name, mode=DEFAULT_MODE, handle=None,
             flags |= _FUNCFLAG_USE_ERRNO
         if use_last_error:
             flags |= _FUNCFLAG_USE_LASTERROR
+        if _sys.platform.startswith("aix"):
+            """When the name contains ".a(" and ends with ")",
+               asin "libFOO.a(libFOO.so)" this is taken to be an
```

Comment:
asin: typo for as in?

Reply:
Yes, I frequently type "asin" as a single world in presentations I give. I'll change it to two words here.